src/Sylius/Bundle/AddressingBundle/Tests/Form/Type/CountryChoiceTypeTest.php

```diff
@@ -13,6 +13,7 @@
 
 namespace Sylius\Bundle\AddressingBundle\Tests\Form\Type;
 
+use Prophecy\PhpUnit\ProphecyTrait;
 use PHPUnit\Framework\Assert;
 use Prophecy\Prophecy\ObjectProphecy;
 use Prophecy\Prophecy\ProphecyInterface;
```

Comment:
The import order is wrong here and in a few other places.
Other than that, looks good.